Solutions/AzureDevOpsAuditing/Data Connectors/mainTemplate.json

```diff
@@ -0,0 +1,132 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "metadata": {
+    "author": "Rogier Dijkman - r.dijkman@securehats.nl",
+    "comments": "Solution template for Azure DevOps Auditing"
+  },
+  "parameters": {
+    "workspace": {
+      "defaultValue": "",
+      "type": "string",
+      "minLength": 1,
+      "metadata": {
+        "description": "Workspace name for Log Analytics where Sentinel is setup"
+      }
+    },
+    "connector1-name": {
+      "type": "string",
+      "defaultValue": "fee94e4b-8505-4c25-b087-ab5fc3edd29f"
+    }
+  },
+  "variables": {
+    "connector1-source": "[concat('/subscriptions/',subscription().subscriptionId,'/resourceGroups/',resourceGroup().name,'/providers/Microsoft.OperationalInsights/workspaces/',parameters('workspace'),'/providers/Microsoft.SecurityInsights/dataConnectors/',parameters('connector1-name'))]",
+    "_connector1-source": "[variables('connector1-source')]"
+  },
+  "resources": [
+    {
+      "id": "[variables('_connector1-source')]",
+      "name": "[concat(parameters('workspace'), '/Microsoft.SecurityInsights/', parameters('connector1-name'))]",
+      "apiVersion": "2021-03-01-preview",
+      "type": "Microsoft.OperationalInsights/workspaces/providers/dataConnectors",
+      "kind": "GenericUI",
+      "properties": {
+        "connectorUiConfig": {
+          "title": " Azure DevOps Auditing",
+          "publisher": "SecureHats",
+          "descriptionMarkdown": "The Azure DevOps Auditing data connector provides the capability to ingest Azure DevOps events with Azure Sentinel. It helps you gain visibility into what is happening in your environment, such as who is connected, which applications are installed and running, and much more.",
```

Comment:
@v-mchatla, please change text from Azure Sentinel to Microsoft Sentinel, thanks.